server.py

```diff
@@ -41,7 +41,13 @@ def index():
             message="Unable to parse input data JSON."
         ), 400
 
-    prediction_worker(input_data, next_service)
+    b64_identity = request.headers['x-rh-identity']
```

Comment:
I would prefer `dict.get` here. Rather safe than sorry..

Reply:
True :)
Sorry for missing that - I just updated the code with that change